src/fxos/readGeckoCommit.js

```diff
@@ -0,0 +1,97 @@
+let assert = require('assert');
+let debug = console.log.bind(console, '[fxos/readGeckoCommit]');
+let denodeify = require('denodeify');
+let fs = require('mz/fs');
+let path = require('path');
+let sax = require('sax');
+let tmpdir = denodeify(require('tmp').dir);
+
+const SOURCES_XML = '/system/sources.xml';
+const APPLICATION_INI = '/system/b2g/application.ini';
+const PLATFORM_INI = '/system/b2g/platform.ini';
+
+function pullIniFile(adb, local, file) {
```

Comment:
You'll see why I went this route, the code for async/await in this case I think makes it look a lot more complex. Let me know if you still want it changed.